joomla/joomla-cms
# [Installation] Fix MySQL sample data for #__menu

### Summary of Changes

This fixes the lft & rgt values for all our sample data. Thanks @infograf768 

### Testing Instructions

- Install https://github.com/zero-24/joomla-cms/archive/fix_broken_menus.zip as normal Joomla
- Install all sample data
- please try that every entry mainitly frontend behaves as expected.

After the installation please clear the tables and do a complete reinstall

### Expected result

Install should work and all menu entrys should work too

### Actual result

After the lastest #__menu changes we had some issues with the frontend menus 

### Documentation Changes Required

None

### Open Points

As i have no Postgress nor Mssql Envoirment this needs to be done by someone else. I'm going to open a new issue for that and link it from here.
